jumpscale/sals/reservation_chatflow/deployer.py

```diff
@@ -253,10 +253,32 @@ def create_pool(self, bot):
         form = bot.new_form()
         cu = form.int_ask("Please specify the required CU", required=True, min=0, default=0)
         su = form.int_ask("Please specify the required SU", required=True, min=0, default=0)
+        time_unit = form.drop_down_choice(
+            "Please choose time unit to use for pool's time-to-live",
+            ["Day", "Month", "Year"],
+            required=True,
+            default="Month",
+        )
+        ttl = form.int_ask(
+            "Please specify the pools time-to-live (result units equals second * UNIT)",
```

Comment:
I think its best to remove the `(result units equals second * UNIT)` as this will likely confuse end users